README.md

```diff
@@ -46,7 +46,7 @@ If you are behind a proxy, configure `https_proxy` environment variable.
 
 ## Built-in starters
 
-- [app](https://github.com/ionic-team/stencil-app-starter)
+- [app](https://github.com/stencil-community/stencil-app-starter)
```

Comment:
possibly we might do this, just to double/triple/quadruple clarify this isn't ionic-maintained code?

```suggestion
- [app (community-maintained)](https://github.com/stencil-community/stencil-app-starter)
```